bluemira/equilibria/fem_fixed_boundary/fem_magnetostatic_2D.py

```diff
@@ -350,8 +351,13 @@ def define_g(
             Target current (also used to initialize the solution in case self.psi is
             still 0 and pprime and ffprime are, then, not defined).
             If None, plasma current is calculated and not constrained
+        B_0: float
+            Toroidal field at R_0 [T]. Used when saving to file.
         """
         self._curr_target = curr_target
+        self._B_0 = B_0
+        self._pprime = pprime
+        self._ffprime = ffprime
```

Comment:
This is only really required so that an EQDSK can be made from a G-S solver. It would actually be much nicer to pass in the raw data of the pprime and ffprime (if they are arrays), and interpolate them internally to the G-S solver. This is because when it comes to saving an equilibrium, there is no easy way of knowing how many points to specify in the psi_norm vector to retrieve the profiles.

The present solution therefore has inaccurate profiles stored in the equilibrium file.